test/system/500-networking.bats

```diff
@@ -400,6 +400,17 @@ load helpers.network
     is "$output" "" "output should be empty"
 }
 
+@test "podman network create with --out will echo its parameter" {
+    is_rootless || skip "only meaningful for rootless"
```

Comment:
It's copy-pasta from another test. Since each of these commands write their result to stdout, I was just adding them for the sake of adding them. But I can totally remove them, it's up to you guys.